configs/MICOAIR743/config.h

```diff
@@ -121,14 +121,13 @@
 #define MAX7456_SPI_INSTANCE SPI1
 #define SDIO_USE_4BIT        1
 #define SDIO_DEVICE          SDIODEV_1
-#define BARO_I2C_INSTANCE    (I2CDEV_2)
-#define MAG_I2C_INSTANCE     (I2CDEV_2)
+#define BARO_I2C_INSTANCE    I2CDEV_2
+#define MAG_I2C_INSTANCE     I2CDEV_2
 #define MAG_I2C_ADDRESS      14
 #define MAG_ALIGN            CW90_DEG
 #define MAG_ALIGN_YAW        900
 
 #define BEEPER_INVERTED
-#define DEFAULT_DSHOT_BITBANG        DSHOT_BITBANG_OFF
```

Comment:
No need to change the DMA requests, just getting rid of this line so that DShot bitbang is used should be sufficient.

Reply:
Have tested dma request changes - as there was no change - it is what we use for latest H7 configs ?